src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -2101,6 +2101,69 @@ internal static BindingRestrictions GetVersionCheck(int expectedVersionNumber)
         }
     }
 
+
+    /// <summary>
+    /// This binder is used to add or remove delegates from event handlers.
+    /// </summary>
+    internal class PSEventDelegateBinder : DynamicMetaObjectBinder
+    {
+        private static readonly PSEventDelegateBinder _addInstanceBinder = new PSEventDelegateBinder(
```

Comment:
Please use `s_` prefix for statics. Below too.
```suggestion
        private static readonly PSEventDelegateBinder s_addInstanceBinder = new PSEventDelegateBinder(
```